### ISSUE TITLE
Create suitable training set (3x3 matrices)

### ISSUE THREAD

**Issue by gomezzz** [esa/neuralg#5]
- Consider gradient explosion avoidance
- Think about the loss a bit (maybe use (input x output - I ) e.g.)
